nemo/collections/llm/gpt/model/llama.py

```diff
@@ -65,11 +65,47 @@ class Llama3Config8B(Llama2Config7B):
     ffn_hidden_size: int = 14336
 
 
+@dataclass
+class Llama3Config8B16K(Llama3Config8B):
+    seq_length: int = 16384
```

Comment:
The parallelism arguments shouldn't typically be set here but in the MegatronStrategy. We try to keep the model configs parallelism agnostic

Reply:
I have changed this PR settings completely based on Hemil's Recipe PR